app/models/user.rb

```diff
@@ -4,4 +4,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :vehicles, dependent: :destroy
+  validates :email, :password, :nick_name, presence: true
```

Comment:
I think it'd be good if we also add a validation for :nick_name, uniqueness: true

Reply:
yes good idee i did it